Create actions-jimo (#6719)
segmentio/segment-docs: src/connections/destinations/catalog/actions-jimo/index.md

> ---
> title: Jimo Destination
> ---

Reviewer: ```suggestion
---
title: Jimo (Actions) Destination
id: 652d4cf5e00c0147e6eaf5e7
beta: true
---
```

{% include content/plan-grid.md name="actions" %}

> [Jimo](https://usejimo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} helps you to publish personalized product experiences to elevate adoption and user engagement without code as a layer on top of your app

Reviewer 2: ```suggestion
[Jimo](https://usejimo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} helps you to publish personalized product experiences to elevate adoption and user engagement without code as a layer on top of your app.
```

This destination is maintained by Jimo. For any issues with the destination, [contact their Support team](mailto:support@usejimo.com).

## Getting started

1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Jimo".
2. Select Jimo and click **Add Destination**.

> 3. Select an existing Source to connect to <Jimo (Actions).

Reviewer 2: ```suggestion
3. Select an existing Source to connect to Jimo (Actions).
```

> 4. Go to the [Jimo dashboard](https://i.usejimo.com/settings/general){:target="_blank"}, find and copy your **project id**.

Reviewer 2: ```suggestion
4. Open your [Jimo dashboard](https://i.usejimo.com/settings/general){:target="_blank"}, find and copy your **project id**.
```

> 5. Enter the **project id** in the Jimo destination settings in Segment.

Reviewer 2: ```suggestion
5. Return to Segment and open the destination settings for the Jimo (Actions) destination that you just created. Enter your Jimo **project id**.
```

> 1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Jimo".
> 2. Select Jimo and click **Add Destination**.
> 3. Select an existing Source to connect to Jimo (Actions).
> 4. Open your [Jimo dashboard](https://i.usejimo.com/settings/general){:target="_blank"}, find and copy your **project id**.
> 5. Return to Segment and open the destination settings for the Jimo (Actions) destination that you just created. Enter your Jimo **project id**.

Reviewer: ```suggestion
1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Jimo (Actions)".
2. Select Jimo (Actions) and click **Add Destination**.
3. Select an existing Source to connect to Jimo (Actions).
4. Open your [Jimo dashboard](https://i.usejimo.com/settings/general){:target="_blank"}, find and copy your **project id**.
5. Return to Segment and open the destination settings for the Jimo (Actions) destination that you just created. Enter your Jimo **project id**.
```
